plugins/woocommerce-admin/client/launch-your-store/hub/sidebar/xstate.tsx

```diff
@@ -222,15 +250,26 @@ export const sidebarMachine = setup( {
 							id: 'prefetch-congrats-data ',
 						} ),
 					],
-					invoke: {
-						src: 'getTasklist',
-						onDone: {
-							actions: assign( {
-								tasklist: ( { event } ) => event.output,
-							} ),
-							target: 'launchYourStoreHub',
+					invoke: [
+						{
+							src: 'getTestOrderCount',
```

Comment:
I'm not sure if this is the right place, but I didn't get an orders count until I re-refreshed the screen. Not sure if that's a timing issue or state staleness.

Reply:
Thank you! I also noticed it, but I thought it was cache issue since refreshing fixed it. It sounds like it's not. I'll try to reproduce it tomorrow.

Reply:
@psealock It was a timing issue. Fixed in [Fix timing issue with counting test orders](https://github.com/woocommerce/woocommerce/pull/47832/commits/dffccf1ebc0f0c42a06ead9862f67bbd87b0af1f)

Reply:
@rjchow I think it's due to the fact that tasklist API is already called on WooCommerce -> Home. It loads instantly from the cache on the Launch your store page and renders the sidebar component before the count call.